services/generic/color_control.md

```diff
@@ -1,45 +1,104 @@
+| `red`           | Yes    | Yes    | 0-255   | Defines intensity of red color.                                                                                                |
+| `green`         | Yes    | Yes    | 0-255   | Defines intensity of green color.                                                                                              |
+| `blue`          | Yes    | Yes    | 0-255   | Defines intensity of blue color.                                                                                               |
+| `warm_w`        | Yes    | No     | 0-255   | Is intensity of warm white light source. Mix of warm white and cold white intensity forms color temperature.                   |
+| `cold_w`        | Yes    | No     | 0-255   | Is intensity of cold white light source. Mix of warm white and cold white intensity forms color temperature.                   |
+| `temp`          | No     | Yes    | 1-65279 | Is the color temperature in Mired (micro reciprocal degree). Actual color temperature supported by end devices is 2700K-6500K. |
+| `amber`         | Yes    | No     | 0-255   | Defines intensity of amber color.                                                                                              |
+| `cyan`          | Yes    | No     | 0-255   | Defines intensity of cyan color.                                                                                               |
+| `purple`        | Yes    | No     | 0-255   | Defines intensity of purple color.                                                                                             |
 
-- cold_w - is cold white light source intensity. Value range 0-255.
+* `transition` is an object representing a single color transition change with the following structure:
 
-- Mix of warm white intensity and cold white intensity forms color temperature.
+| Field      | Type   | Example | Description                                         |
+|------------|--------|---------|-----------------------------------------------------|
+| component  | string | `"red"` | One of the supported `color_component` values.      |
+| transition | string | `"up"`  | Direction of the transition, either `up` or `down`. |
+| duration   | int    | `127`   | Optional duration of the transition in seconds.     |
 
-- Duration property within cmd.color.start_transition command is optional and means seconds.
+* `transition_range` is an object with the following structure:
 
-- In the Z-Wave protocol, color component transition duration can only be set between 1 - 127 seconds or 1 - 127 minutes. In order to fully support the Z-Wave protocol, sup_durations property can contain an array of ranges with a specific step.
+| Field | Type | Example | Description                        |
+|-------|------|---------|------------------------------------|
+| min   | int  | `1`     | Minimum settable value in seconds. |
```

Comment:
Example is not helpful. Supported values with give more insight: like for min <1,max>,
for max woudl be <min, 127> etc.

Reply:
Please read this spec more carefully. Transition range object is part linked from sup_durations READ-ONLY service property. It's adapter who provides you this structure for you as a suer to figure at what are min max values. You comment does not make sense to me here.